/**
 * The interface for a minimalistic service that returns the quantity for a
 * given item_id.
 */
syntax = "proto3";

package store;

service Store {
  rpc GetQuantity(QuantityRequest) returns (QuantityResponse) {}
  rpc StocksItems(stream Item) returns (Bool) {}
}

message Item {
  int32 item_id = 1;
}

message Bool {
  bool value = 1;
}

message QuantityRequest {
  int32 item_id = 1;
}

message QuantityResponse {
  int32 quantity = 1;
}
